src/osd/OSD.cc

```diff
@@ -1457,22 +1457,19 @@ void OSDService::send_incremental_map(epoch_t since, Connection *con,
            << " to " << con << " " << con->get_peer_addr() << dendl;
 
   MOSDMap *m = NULL;
```

Comment:
you should probably avoid initializing 'm' (and surely not to null).

Reply:
Other than that - looks good

Reply:
@ronen-fr Why do you want this to be uninitialized?  Ideally, we'd simply initialize it to the correct value, but failing that I really don't want it left uninitialized -- if the below if/else is incorrectly modified to fail to set it it a null value will be far easier to debug.

Reply:
Converting the if/else to a lambda that returns m into send_map would, of course, eliminate the problem.